python/delta_sharing/reader.py

```diff
@@ -93,6 +93,22 @@ def to_pandas(self) -> pd.DataFrame:
             copy=False,
         )[[field["name"] for field in schema_json["fields"]]]
 
+    def table_changes_to_pandas(self, cdfOptions: CdfOptions) -> pd.DataFrame:
+        response = self._rest_client.list_table_changes(self._table, cdfOptions)
+
+        schema_json = loads(response.metadata.schema_string)
+
+        if len(response.actions) == 0 or self._limit == 0:
+            return get_empty_table(schema_json)
+
+        converters = to_converters(schema_json)
+        pdfs = []
+        for action in response.actions:
```

Comment:
Are we able to tell the class of these actions? For 'AddFile' and 'RemoveFile', we need to manually add column `_change_type`, as 'insert' or 'remove'.

Reply:
Made changes to overwrite the change type as discussed.